external/loaders/tests/test__batch.py

```diff
@@ -286,3 +286,22 @@ def test_batches_from_netcdf(tmpdir):
     )
     for ds_saved, ds_loaded in zip(saved_batches, loaded_batches):
         xr.testing.assert_equal(ds_saved, ds_loaded)
+
+
+def test_batches_from_mapper_stacked_data_is_not_shuffled():
```

Comment:
Does the test `test_batches_from_mapper_stacked_data_is_shuffled` need to be modified/deleted?

Reply:
Since I kept `shuffle_samples` and `shuffle_timesteps` as options, I didn't change that test.